docs/scripts/api/buildComponentsDocumentation.ts

```diff
@@ -183,19 +177,57 @@ function parseComponentSource(src: string, componentObject: { filename: string }
   return reactAPI;
 }
 
+function findXDemos(
+  componentName: string,
+  pagesMarkdown: ReadonlyArray<PageMarkdown>,
+): ReactApi['demos'] {
+  if (componentName.startsWith('Grid') || componentName.startsWith('DataGrid')) {
+    const demos: ReactApi['demos'] = [];
+    if (componentName === 'DataGrid' || componentName.startsWith('Grid')) {
+      demos.push({ name: 'DataGrid', demoPathname: '/x/react-data-grid/#mit-version' });
+    }
+    if (componentName === 'DataGridPro' || componentName.startsWith('Grid')) {
+      demos.push({ name: 'DataGridPro', demoPathname: '/x/react-data-grid/#commercial-version' });
+    }
+    if (componentName === 'DataGridPremium' || componentName.startsWith('Grid')) {
+      demos.push({
+        name: 'DataGridPremium',
+        demoPathname: '/x/react-data-grid/#commercial-version',
+      });
+    }
+
+    return demos;
+  }
+
+  return pagesMarkdown
+    .filter((page) => {
+      if (!page.pathname.includes('/')) {
+        return false;
+      }
+
+      return page.pathname.indexOf('/x') && page.components.includes(componentName);
```

Comment:
Copy pasted from the core :laughing: 

Their, they check if the link is a `base` or a `joy` or a `material`. To avoid having the `/joy/button` page matching the Material Button API page.
But for X it's probably useless indeed...

Reply:
Done